client/blocks/login/login-form.jsx

```diff
@@ -413,6 +414,7 @@ export class LoginForm extends Component {
 			socialAccountIsLinking: linkingSocialUser,
 			isJetpackWooCommerceFlow,
 			isGutenboarding,
+			isAnchorFmSignup,
```

Comment:
Even though I'm not a fan of `propTypes` generally, seems like it would be best to add this to the `propTypes` of this component to keep that up to date e.g.

```
isAnchorFmSignup: PropTypes.bool
```

Reply:
Thanks, added to both components in https://github.com/Automattic/wp-calypso/pull/50285/commits/1a6cb1d869e7e207f1b72a4eff06de48f8bc8c9a